addon/helpers/object.js

```diff
@@ -0,0 +1,43 @@
+import { MicroState } from 'ember-microstates';
+import assign from '../utils/assign';
+import { reduceObject } from '../utils/object-utils';
+
+export default MicroState.extend({
+
+  prototypeFor(value) {
+    return assign({}, value);
```

Comment:
Is there any reason to clone the object? It seems like an extra level of indirection.